New guide for custom metric governance (#24608)
DataDog/documentation: content/en/account_management/billing/custom_metrics.md

aliases:
    - /integrations/faq/what-standard-integrations-emit-custom-metrics/
further_reading:
- link: "/metrics/custom_metrics/"
  tag: "Documentation"
  text: "Learn more about Custom Metrics"
- link: "/metrics/guide/custom_metrics_governance/"
  tag: "Guide"
  text: "Best Practice for Custom Metric Governance"
algolia:
  tags: ['custom metrics billing']
---

## Overview

If a metric is not submitted from one of the [more than {{< translate key="integration_count" >}} Datadog integrations][1] it's considered a [custom metric][2]. Certain standard integrations can also potentially emit custom metrics. For more information, see [Custom metrics and standard integrations][14].

**A custom metric is uniquely identified by a combination of a metric name and tag values (including the host tag)**. In general, any metric you send using [DogStatsD][3] or through a [custom Agent Check][4] is a custom metric.

Your monthly billable count for custom metrics (reflected on the Usage page) is calculated by taking the total of all distinct custom metrics for each hour in a given month, and dividing it by the number of hours in the month to compute a monthly average value.

> **Custom metrics usage is billed solely on tag value combinations** (also referred to as "metric timeseries", not to be confused with timeseries visualizations). You can send metrics to Datadog without controlling metric submission frequency or the number of queries you run on your metrics. You are not penalized for submitting more granular values on your metrics or trying to get more value from the data you submit to Datadog. 

Author: @ijkaylin I reworded your original paragraph on single dimension billing and added it to the custom metrics billing doc. Can you check the wording to make sure I didn't misinterpret the meaning of this?

> **Custom metrics usage is billed solely on tag value combinations** (also referred to as "metric timeseries", not to be confused with timeseries visualizations). You can send metrics to Datadog without controlling metric submission frequency or the number of queries you run on your metrics. You are not penalized for submitting more granular values on your metrics or trying to get more value from the data you submit to Datadog. 

Reviewer: ```suggestion
**Custom metrics usage is billed solely on tag value combinations**. (Custom metrics is also referred to as *metric timeseries*, and is not to be confused with timeseries visualizations.) You can send metrics to Datadog without controlling metric submission frequency or the number of queries you run on your metrics. You are not penalized for submitting more granular values on your metrics or trying to get more value from the data you submit to Datadog. 
```

Reviewer: tried to help the first sentence not carry so much info. Not 100% sure that my suggestion, `Custom metrics is also referred to as *metric timeseries*`, is correct.

Author: Timeseries is the tag value combinations not custom metrics, but I'll swap out the two terms. Your suggestion to break it up makes it less confusing!

> **Custom metrics usage is billed solely on tag value combinations** (also referred to as "metric timeseries", not to be confused with timeseries visualizations). You can send metrics to Datadog without controlling metric submission frequency or the number of queries you run on your metrics. You are not penalized for submitting more granular values on your metrics or trying to get more value from the data you submit to Datadog. 

Author: ```suggestion
Your monthly billable custom metrics usage (reflected on the Usage page) is calculated by taking the total of all distinct custom metrics (also known as timeseries) for each hour in a given month, and dividing it by the number of hours in the month to compute a monthly average value. Your billable usage is not impacted by data point submission frequency or the number of queries you run on your metrics.
```

> Your monthly billable count for custom metrics (reflected on the Usage page) is calculated by taking the total of all distinct custom metrics for each hour in a given month, and dividing it by the number of hours in the month to compute a monthly average value.
> 
> **Custom metrics usage is billed solely on the metric time series**. (Metric time series are also referred to as *tag value combinations*, and are not to be confused with timeseries visualizations.) You can send metrics to Datadog without controlling metric submission frequency or the number of queries you run on your metrics. You are not penalized for submitting more granular values on your metrics or trying to get more value from the data you submit to Datadog. 

Author: ```suggestion
Your monthly billable custom metrics usage (reflected on the Usage page) is calculated by taking the total of all distinct custom metrics (also known as timeseries) for each hour in a given month, and dividing it by the number of hours in the month to compute a monthly average value. Your billable usage is not impacted by data point submission frequency or the number of queries you run on your metrics.
```
